<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-G735JHSB" name="com.hrms.testscripts.TC01" tests="1" failures="1" timestamp="3 Feb 2021 16:26:03 GMT" time="0.076" errors="0">
  <testcase name="tc01" time="0.076" classname="com.hrms.testscripts.TC01">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The type com.hrms.utility.Log is not visible
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	The type com.hrms.utility.Log is not visible
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved
	Log cannot be resolved

	at com.hrms.lib.General.<init>(General.java:8)
	at com.hrms.testscripts.TC01.tc01(TC01.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- tc01 -->
</testsuite> <!-- com.hrms.testscripts.TC01 -->
